content/pages/features.html

```diff
@@ -20,7 +20,7 @@ <h3>A central dashboard to start your day</h3>
                 </div>
 
                 <div class="w50" style="float: left;">
-                    <div class="about-panel-title">For test engineers</div>
+                    <div class="about-panel-title">For test enginners</div>
```

Comment:
Somehow the rebase didn't work for you but the latest source branch has this typo fixed.